src/schema/article-renderer.tsx

```diff
@@ -171,8 +171,10 @@ function renderElement(props: RenderElementProps): React.ReactNode {
   const { element, children, path } = props
 
   if (element.type === 'a') {
+    const isOnProfile =
+      path && typeof path[0] === 'string' && path[0].startsWith('profile')
```

Comment:
Out of curiosity: Where is path set?

Reply:
https://github.com/serlo/frontend/blob/staging/src/components/pages/user/profile.tsx#L90
It's a parameter of `renderArticle`

Reply:
quite handy feature, used it a lot in #1133